src/lib/Libutil/misc_utils.c

```diff
@@ -657,64 +657,30 @@ copy_file_internal(char *src, char *dst)
  */
 
 int
-lock_file(FILE *fp, int op, char *filename, int lock_retry,
+lock_file(int fd, int op, char *filename, int lock_retry,
 	char *err_msg, size_t err_msg_len)
```

Comment:
Using log_eventf here will add dependency of liblog in libutil, I think current signature is right way to do, it will make this func independent of any dependency